docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -21,6 +27,35 @@ To simplify setting up Rancher for installing Rancher Turtles, the official Ranc
 - Disables the `embedded-cluster-api` feature in Rancher.
 - Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks, as they are no longer needed.
 
+The above webhooks can be removed through the Rancher UI as well and are accessed from the left-hand navigation from your local cluster under the **More Resources** > **Admission** section or found by using the **Resource Search** field and inputting the webhook names. Additionally, the following `kubectl` commands can also be used to remove the necessary webhooks:
```

Comment:
```suggestion
These webhooks can be removed through the Rancher UI as well:

1. In the upper left corner, click **☰** > **Cluster Management**.
1. Select your local cluster.
1. In the left-hand navigation menu, select **More Resources** > **Admission**.

They can also be accessed by entering the names of the webhooks into the **Resource Search** field.

The following `kubectl` commands can manually remove the necessary webhooks:
```

The instructions need one more step describing what to click to remove the webhooks after the user accesses Admission.

Reply:
Added a couple more steps to clarify where the "Delete" option can be selected along with your suggestion. Let me know if it makes sense, thanks!